src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +19,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(sessionFactory);
```

Comment:
With the changes I suggested it should be done like this now:

```suggestion
        CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));
```